xlsxwriter/worksheet.py

```diff
@@ -350,6 +350,38 @@ def __init__(self):
         self.vertical_dpi = 0
         self.horizontal_dpi = 0
 
+    def write_token_as_string(self, token, row, col, *args):
```

Comment:
Merged as is, apart from whitespace and comment changes, and the minor change below.